website/content/docs/api-gateway/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+layout: docs
+page_title: Consul API Gateway Overview
+description:  >-
+  Using Consul API gateway functionality
+---
+
+# Consul API Gateway Overview
+
+This topic provides an overview of the Consul API Gateway. 
+
+## What is Consul API Gateway?
+
+Consul API Gateway is an add-on for Consul that helps users control access to services running within a Consul service mesh. The API gateway enables external network clients to access applications and services running in a Consul datacenter. This type of network traffic is commonly referred to as "north-south" network traffic as it refers to the flow of data into and out of a specific environment. Requests from clients can also be forwarded based on path or request protocol.
+
+Consul API Gateway solves the following primary use cases:
+
+- **Controlling access at the point of entry**: Consul API Gateway allows users to set the protocols of external connection requests and provide clients with TLS certificates from trusted providers (e.g., Verisign, Let’s Encrypt).
+- **Simplifying traffic management**: The Consul API Gateway can load balance requests across services and route traffic to the appropriate service by matching one or more criteria, such as hostname, path, header presence or value, and HTTP Method type (e.g., GET, POST, PATCH).
+
+## Implementation
+
+Consul API Gateway can be deoployed, currently, only on Kubernetes-based runtime environments. Addional runtimes will be supported In the future. It also requires Consul Service Mesh be deployed on the Kubernetes cluster.
+
+API Gateway can route traffic to services connected to the same service mesh its connected to. Those services can be running on the Kubernetes cluster or somewhere else, if they are connected to the same service mesh.
```

Comment:
```suggestion
API Gateway routes traffic to services connected to the same service mesh. If the services can run on the Kubernetes cluster or another location if they are connected to the same service mesh.
```
Just trying to see if we can make this clearer. Can we provide more concrete language for "somewhere else" in this description? I said "another location", but maybe we mean "a cluster hosted on another runtime"?

Reply:
```suggestion
API Gateway routes traffic to services connected to the same service mesh. Those services can be running on the same Kubernetes cluster as the API Gateway or somewhere else, as long as they are connected to the same service mesh deployment and reachable over the network.
```